### PR TITLE
write comment for view didload

### PR DESCRIPTION
**Pull request by Harrisali93**
this is adding a comment on view didload

### CODE REVIEW

File: RedditNewsFeed/ViewControllers/RedditNewsFeedViewController.swift [hunk 1]
```diff
@@ -19,12 +19,13 @@ class RedditNewsFeedViewController: UIViewController {
     let child = SpinnerViewController()
     
     var isValidKey:Bool = false
-    
+    //this is view didLoad
```
Comment (arfhan786) [inline RedditNewsFeed/ViewControllers/RedditNewsFeedViewController.swift:23]: Please add one space above the comment

File: RedditNewsFeed/ViewControllers/RedditNewsFeedViewController.swift [hunk 2]
```diff
@@ -19,12 +19,13 @@ class RedditNewsFeedViewController: UIViewController {
...
     override func viewDidLoad() {
         super.viewDidLoad()
         configureUI()
         getNewsFeeds(after: "10")
         view.backgroundColor = .red
+
```
Comment (arfhan786) [inline RedditNewsFeed/ViewControllers/RedditNewsFeedViewController.swift]: please remove this space

File: RedditNewsFeed/ViewControllers/RedditNewsFeedViewController.swift (hunk 1 above)
Comment (Harrisali93) [reply RedditNewsFeed/ViewControllers/RedditNewsFeedViewController.swift:23]: resolve

File: RedditNewsFeed/ViewControllers/RedditNewsFeedViewController.swift (hunk 2 above)
Comment (Harrisali93) [reply RedditNewsFeed/ViewControllers/RedditNewsFeedViewController.swift]: resolve

**Review by arfhan786 — APPROVED** [review]

**Review by RiffatiKhan — APPROVED** [review]

**Pull request merged**